src/dbnode/storage/index.go

```diff
@@ -1688,12 +1628,12 @@ func (i *nsIndex) queryWithSpan(
 	i.metrics.querySearchTime.ByRange.Record(queryRange, results.TotalDuration().Search)
 	i.metrics.querySearchTime.ByDocs.Record(results.TotalDocsCount(), results.TotalDuration().Search)
 
-	return exhaustive, nil
+	return exhaustive, err
 }
 
 func (i *nsIndex) execBlockQueryFn(
 	ctx context.Context,
-	cancellable *xresource.CancellableLifetime,
+	_ *xresource.CancellableLifetime,
```

Comment:
Should we just remove these from the exec block query fn's since they are unused now? Or is this just here until aggregate does not use cancellable anymore?

Reply:
removed since I updated the aggregate path as well